pkg/block/fetcher.go

```diff
@@ -185,7 +184,7 @@ func (s *MetaFetcher) loadMeta(ctx context.Context, id ulid.ULID) (*metadata.Met
 			return m, nil
 		}
 
-		if !stderrors.Is(err, os.ErrNotExist) {
+		if !errors.Is(err, os.ErrNotExist) {
```

Comment:
FYI: This actually works thanks to: https://github.com/pkg/errors/commit/6d954f502eb89cd315e4baae5b0e0db516d6f787